client/components/Details/PageTypes/Cell/Experimental.js

```diff
@@ -96,26 +109,25 @@ function Hero({ instance }) {
           <MorphologyPreview onHover={() => {}} value={instance} shouldRender />
         </picture>
       </div>
-      {files &&
-        files.length && (
-          <div className="detail-attachments">
-            <h3>
-              <Icon type="paper-clip" />{" "}
-              <Download
-                files={files}
-                name={getProp(instance, "cellName.label", "Cell")}
-              >
-                <a>
-                  {files.length} File
-                  {files.length > 1 ? "s" : ""}
-                </a>
-              </Download>
-            </h3>
-            <ul>
-              <li />
-            </ul>
-          </div>
-        )}
+      {files && files.length && (
+        <div className="detail-attachments">
+          <h3>
+            <Icon type="paper-clip" />{" "}
+            <Download
+              files={files}
+              name={getProp(instance, "cellName.label", "Cell")}
+            >
+              <a>
+                {files.length} File
+                {files.length > 1 ? "s" : ""}
```

Comment:
OK for this lone occurrence, but that makes me wonder if on the long term we should consider integrating something like https://github.com/yahoo/react-intl

Reply:
Let's consider this for nexus web. I've created an [issue #297 for that ](https://github.com/BlueBrain/nexus/issues/297)